modules/openapi-generator/src/main/resources/scala-sttp-client/api.mustache

```diff
@@ -0,0 +1,65 @@
+{{>licenseInfo}}
+package {{package}}
+
+{{#imports}}
+import {{import}}
+{{/imports}}
+import {{mainPackage}}.core._
+import {{mainPackage}}.core.CollectionFormats._
+import sttp.client._
+import sttp.model.Method
+import sttp.client.json4s.SttpJson4sApi
+
+{{#operations}}
+object {{classname}} {
+
+  def apply(baseUrl: String = "{{{basePath}}}")(implicit serializer: SttpSerializer) = new {{classname}}(baseUrl)
+}
+
+class {{classname}}(baseUrl: String)(implicit serializer: SttpSerializer) {
+
+  import serializer._
+
+  /*
+   * Helper to handle Optional header parameters
+   **/
+  implicit class optionalParams(request: RequestT[Identity, Either[String, String], Nothing]) {
+   def header( header: String, optValue: Option[Any]): RequestT[Identity, Either[String, String], Nothing] = {
+      optValue.map( value => request.header(header, value.toString)).getOrElse(request)
+    }
+  }
+
+{{#operation}}
+{{#javadocRenderer}}
+{{>javadoc}}
+{{/javadocRenderer}}
+  def {{operationId}}({{>methodParameters}}): RequestT[Identity, Either[ResponseError[Exception], {{>operationReturnType}}], Nothing] =
+    basicRequest
+      .method(Method.{{httpMethod.toUpperCase}}, uri"$baseUrl{{{path}}}{{#queryParams.0}}?{{#queryParams}}{{baseName}}=${{{paramName}}}{{^-last}}&{{/-last}}{{/queryParams}}{{/queryParams.0}}")
+      .contentType({{#consumes.0}}"{{{mediaType}}}"{{/consumes.0}}{{^consumes}}"application/json"{{/consumes}}){{#headerParams}}
```

Comment:
there must be string constant from `sttp` for `"application/json"`

Reply:
this seems to long `sttp.model.MediaType.ApplicationJson.toString()`